CBDatabase/Protocols/DatabaseModelObject.swift

```diff
@@ -12,7 +12,30 @@ import CoreData
 ///         - Add the entity's attributes as ivars.
 ///         - If the entity name differs from the name of the struct, be sure to override entityName.
 public protocol DatabaseModelObject: Codable, Hashable {
+    static var entityName: String { get }
+}
+
+public protocol IdentifiableDatabaseModelObject: DatabaseModelObject {
```

Comment:
since we're breaking compatibility with Wallet's models. I always wanted to rename this protocol to something shorter.

Can we rename `DatabaseModelObject` to `DBModel` and `IdentifiableDatabaseModelObject` to `IdentifiableDBModel`?

Reply:
Yup!